res/values/setup.xml

```diff
@@ -28,6 +28,7 @@
     
     
     <!-- Colors -->
+    <color name="login_text_color">#757575</color>
```

Comment:
If the login_text_color is moved to setup.xml shouldn't the login_text_hint_color also being moved here, or rather reference login_text_color to make sure it the hint color matched the text color? :)